src/measure_theory/lp_space.lean

```diff
@@ -1678,3 +1730,60 @@ end Lp
 end measure_theory
 
 end complete_space
+
+namespace bounded_continuous_function
+
+open_locale bounded_continuous_function
+variables [borel_space E] [second_countable_topology E]
+variables [topological_space α] [borel_space α]
+variables [finite_measure μ]
+
+lemma mem_Lp (f : α →ᵇ E) :
+  f.to_continuous_map.to_ae_eq_fun μ ∈ Lp E p μ :=
+begin
+  refine Lp.mem_Lp_of_ae_bound (∥f∥) _,
+  filter_upwards [f.to_continuous_map.coe_fn_to_ae_eq_fun μ],
+  intros x hx,
+  convert f.norm_coe_le_norm x
+end
+
+/-- The normed group homomorphism of considering a bounded continuous function on a finite-measure
+space as an element of `Lp`. -/
+def to_Lp_hom [fact (1 ≤ p)] : normed_group_hom (α →ᵇ E) (Lp E p μ) :=
+{ bound' := ⟨(measure_univ_nnreal μ) ^ (p.to_real)⁻¹, λ f, begin
+    apply Lp.norm_le_of_ae_bound (norm_nonneg f),
+    { filter_upwards [f.to_continuous_map.coe_fn_to_ae_eq_fun μ],
+      intros x hx,
+      convert f.norm_coe_le_norm x },
+    { apply_instance }
+  end⟩,
+  .. add_monoid_hom.cod_restrict
+      ((continuous_map.to_ae_eq_fun_add_hom μ).comp (forget_boundedness_add_hom α E))
+      (Lp E p μ)
+      mem_Lp }
+
+variables (𝕜 : Type*) (E p μ)
+
+/-- The bounded linear map of considering a bounded continuous function on a finite-measure space
+as an element of `Lp`. -/
+def to_Lp [normed_field 𝕜] [normed_space 𝕜 E] [fact (1 ≤ p)] : (α →ᵇ E) →L[𝕜] (Lp E p μ) :=
+linear_map.mk_continuous
+  (linear_map.cod_restrict
+    (Lp.Lp_submodule E p μ 𝕜)
+    ((continuous_map.to_ae_eq_fun_linear_map μ).comp (forget_boundedness_linear_map α E))
+    mem_Lp)
+  ((measure_univ_nnreal μ) ^ (p.to_real)⁻¹)
+  begin
+    intros f,
+    apply Lp.norm_le_of_ae_bound (norm_nonneg f),
+    { filter_upwards [f.to_continuous_map.coe_fn_to_ae_eq_fun μ],
+      intros x hx,
+      convert f.norm_coe_le_norm x },
+    { apply_instance }
```

Comment:
You are duplicating a proof in `to_Lp_hom`. Can you factor it out as a lemma?

Reply:
Thanks, indeed that's shorter!